plugins/opusCodec/src/OpusEncoder.cpp

```diff
@@ -0,0 +1,269 @@
+}
+
+void AthenaOpusEncoder::setVBRConstraint(int vbr_const)
+{
+    assert(_encoder);
+    int ret = opus_encoder_ctl(_encoder, OPUS_SET_VBR_CONSTRAINT(vbr_const));
+    if ( ret != OPUS_OK )
+        qCWarning(encoder) << "Error when setting VBR constraint to " << vbr_const << ": " << error_to_string(ret);
+}
+
+int AthenaOpusEncoder::getMaxBandwidth() const
+{
+    assert(_encoder);
+    int ret;
+    opus_encoder_ctl(_encoder, OPUS_GET_MAX_BANDWIDTH(&ret));
+    return ret;
+}
+
+void AthenaOpusEncoder::setMaxBandwidth(int maxbw)
+{
+    assert(_encoder);
+    int ret = opus_encoder_ctl(_encoder, OPUS_SET_MAX_BANDWIDTH(maxbw));
+    if ( ret != OPUS_OK )
+        qCWarning(encoder) << "Error when setting max bandwidth to " << maxbw << ": " << error_to_string(ret);
+}
+
+int AthenaOpusEncoder::getBandwidth() const
+{
+    assert(_encoder);
+    int ret;
+    opus_encoder_ctl(_encoder, OPUS_GET_BANDWIDTH(&ret));
+    return ret;
+}
+
+void AthenaOpusEncoder::setBandwidth(int bw)
+{
+    assert(_encoder);
+    int ret = opus_encoder_ctl(_encoder, OPUS_SET_BANDWIDTH(bw));
+    if ( ret != OPUS_OK )
+        qCWarning(encoder) << "Error when setting bandwidth to " << bw << ": " << error_to_string(ret);
+}
+
+int AthenaOpusEncoder::getSignal() const
+{
+    assert(_encoder);
+    int ret;
+    opus_encoder_ctl(_encoder, OPUS_GET_SIGNAL(&ret));
+    return ret;
+}
+
+void AthenaOpusEncoder::setSignal(int signal)
+{
+    assert(_encoder);
+    int ret = opus_encoder_ctl(_encoder, OPUS_SET_SIGNAL(signal));
+    if ( ret != OPUS_OK )
+        qCWarning(encoder) << "Error when setting signal to " << signal << ": " << error_to_string(ret);
+}
+
+int AthenaOpusEncoder::getApplication() const
+{
+    assert(_encoder);
+    int ret;
+    opus_encoder_ctl(_encoder, OPUS_GET_APPLICATION(&ret));
+    return ret;
+}
+
+void AthenaOpusEncoder::setApplication(int app)
+{
+    assert(_encoder);
+    int ret = opus_encoder_ctl(_encoder, OPUS_SET_APPLICATION(app));
+    if ( ret != OPUS_OK )
+        qCWarning(encoder) << "Error when setting application to " << app << ": " << error_to_string(ret);
+}
+
+int AthenaOpusEncoder::getLookahead() const
+{
+    assert(_encoder);
+    int ret;
+    opus_encoder_ctl(_encoder, OPUS_GET_LOOKAHEAD(&ret));
+    return ret;
+}
+
+int AthenaOpusEncoder::getInbandFEC() const
+{
+    assert(_encoder);
+    int ret;
+    opus_encoder_ctl(_encoder, OPUS_GET_INBAND_FEC(&ret));
+    return ret;
+}
+
+void AthenaOpusEncoder::setInbandFEC(int fec)
+{
+    assert(_encoder);
+    int ret = opus_encoder_ctl(_encoder, OPUS_SET_INBAND_FEC(fec));
+    if ( ret != OPUS_OK )
+        qCWarning(encoder) << "Error when setting inband FEC to " << fec << ": " << error_to_string(ret);
+}
+
+int AthenaOpusEncoder::getExpectedPacketLossPercent() const
+{
+    assert(_encoder);
+    int ret;
+    opus_encoder_ctl(_encoder, OPUS_GET_PACKET_LOSS_PERC(&ret));
+    return ret;
+}
+
+void AthenaOpusEncoder::setExpectedPacketLossPercent(int perc)
+{
```

Comment:
Please put { on same line as function name  CS 4.1.4 https://github.com/kasenvr/project-athena/blob/kasen/core/CODING_STANDARD.md#414-method-definitions-should-have-the-following-form